training/vision/components/image_classification/spec.yaml

```diff
@@ -79,23 +176,40 @@ outputs:
 
 code: ../src
 
-environment: azureml:AzureML-AutoML-DNN-Vision-GPU:98
+environment: azureml:AzureML-AutoML-DNN-Vision-GPU:107
```

Comment:
@sharma-riti Are we expected to change this after every CE release?

Reply:
We are still looking into the release process... for now wanted to upgrade to the latest CE... 

I am not very familiar with https://github.com/Azure/azureml-assets/tree/main/training/automl/components components and how they work ... maybe I'll sync with you and @novaturient95 to learn more and see if these are duplicates ...

Reply:
Updated it to use the latest always.

Reply:
Also, synced with @novaturient95 , these ones don't run the dpv2 flow but just the runtime code, so both the components are different.